graphql/project_resolver.go

```diff
@@ -42,7 +42,7 @@ func (r *projectResolver) Patches(ctx context.Context, obj *restModel.APIProject
 	apiPatches := []*restModel.APIPatch{}
 	for _, p := range patches {
 		apiPatch := restModel.APIPatch{}
-		err = apiPatch.BuildFromService(p, nil) // Injecting DB info into APIPatch is handled by the resolvers.
+		err = apiPatch.BuildFromService(p, &restModel.APIPatchArgs{}) // Injecting DB info into APIPatch is handled by the resolvers.
```

Comment:
why? we should make sure its okay to pass in nil here

Reply:
removed